web/packages/teleterm/src/ui/DocumentTerminal/DocumentTerminal.tsx

```diff
@@ -62,59 +117,15 @@ export function DocumentTerminal(props: Props & { visible: boolean }) {
       onContextMenu={state.data?.openContextMenu}
       autoFocusDisabled={true}
     >
+      {$fileTransfer}
       {ptyProcess && (
-        <>
-          {doc.kind === 'doc.terminal_tsh_node' && (
```

Comment:
Previously, the file transfer UI would be shown only after the PTY got created. This doesn't work for the doc type as the PTY is created only after `loginHost` is resolved.